package.json

```diff
@@ -16,7 +16,7 @@
     "test": "grunt test --stack"
   },
   "dependencies": {
-    "grunt-lib-phantomjs": "^1.0.0"
+    "grunt-lib-phantomjs": "^1.1.0"
```

Comment:
`^1.0.0` is equivalent to `>=1.0.0 & <2.0.0`, so no need to update the version. `npm` will install version `1.1.0` automatically.